frontend/src/app/projects/dashboard/metrics/page.tsx

```diff
@@ -0,0 +1,283 @@
+    {
+      title: 'Project Level',
+      items: [
+        { label: 'Incubator', key: 'incubator' },
+        { label: 'Lab', key: 'lab' },
+        { label: 'Production', key: 'production' },
+        { label: 'Flagship', key: 'flagship' },
+      ],
+    },
+    {
+      title: 'Project Health',
+      items: [
+        { label: 'Healthy Projects', key: 'healthy' },
+        { label: 'Projects Needing Attention', key: 'needsAttention' },
+        { label: 'Unhealthy Projects', key: 'unhealthy' },
+      ],
+    },
+    {
+      title: 'Reset Filters',
+      items: [{ label: 'Reset All Filters', key: 'reset' }],
+    },
+  ]
+
+  const orderingMapping = {
+    scoreDESC: {
+      scoreOrdering: { score: 'DESC' },
+    },
+    scoreASC: {
+      scoreOrdering: { score: 'ASC' },
+    },
+    starsCountDESC: {
+      starsCountOrdering: { starsCount: 'DESC' },
+    },
+    starsCountASC: {
+      starsCountOrdering: { starsCount: 'ASC' },
+    },
+    forksCountDESC: {
+      forksCountOrdering: { forksCount: 'DESC' },
+    },
+    forksCountASC: {
+      forksCountOrdering: { forksCount: 'ASC' },
+    },
+    contributorsCountDESC: {
+      contributorsCountOrdering: { contributorsCount: 'DESC' },
+    },
+    contributorsCountASC: {
+      contributorsCountOrdering: { contributorsCount: 'ASC' },
+    },
+  }
+
+  const getCurrentPage = () => {
+    return Math.floor(pagination.offset / PAGINATION_LIMIT) + 1
+  }
+
+  return (
+    <>
+      <div className="mb-4 flex items-center justify-between">
+        <h1 className="text-2xl font-bold">Project Health Metrics</h1>
+        <div className="flex items-center space-x-2">
+          <ProjectsDashboardDropDown
+            buttonDisplayName="Filter By"
+            icon={faFilter}
+            sections={filteringSections}
+            selectionMode="single"
+            selectedKeys={[activeFilterKey]}
+            onAction={async (key: string) => {
+              // Because how apollo caches pagination, we need to reset the pagination.
+              const newPagination = { offset: 0, limit: PAGINATION_LIMIT }
+              setPagination(newPagination)
+              setFilters(filtersMapping[key])
+              setActiveFilterKey(key)
+              await refetch({
+                filters: filtersMapping[key],
+                pagination: newPagination,
+                ordering: Object.values(ordering),
+              })
+              const paramsMapping = {
+                healthy: '?health=healthy',
+                needsAttention: '?health=needsAttention',
+                unhealthy: '?health=unhealthy',
+                reset: '',
+                incubator: '?level=incubator',
+                lab: '?level=lab',
+                production: '?level=production',
+                flagship: '?level=flagship',
+              }
+              router.push(`/projects/dashboard/metrics${paramsMapping[key] || ''}`)
+            }}
+          />
+
+          <ProjectsDashboardDropDown
```

Comment:
We have a `SortBy` component already that is pretty customizable. Do you think we could use it here for style consistency? 🤔 
<img width="959" height="303" alt="Screenshot 2025-07-13 at 4 13 31 PM" src="https://github.com/user-attachments/assets/0db0189d-083b-4edf-8343-8aa7677c9038" />